src/FileConventions/Library.fs

```diff
@@ -104,3 +104,39 @@ let EolAtEof(fileInfo: FileInfo) =
                 False
         else
             True
+
+let allowedNonVerboseFlags = seq { "env -S" }
```

Comment:
@realmarv we need to add "unzip" here, because it doesn't have long flags

Reply:
(let's do it with a new unit test too)

Reply:
Done and rebased